code/game/objects/items/robot/robot_parts.dm

```diff
@@ -11,25 +11,25 @@
 	name = "robot left arm"
 	desc = "A skeletal limb wrapped in pseudomuscles, with a low-conductivity case."
 	icon_state = "l_arm"
-	part = list("l_arm", "l_hand")
+	part = list(ARM_LEFT, HAND_LEFT)
```

Comment:
These are clothing protection bitflags. Check defines/mobs.dm for BODY_ZONE ones instead.

Reply:
Wait, limbs actually use these ones for surgery/body_part? That's weird. Disregard me then.

Reply:
I've modified the part list to use the correct BODY_ZONE defines, and for the surgery code to check the  limb name instead.